src/utils/neurons.utils.ts

```diff
@@ -0,0 +1,21 @@
+import { Principal } from "@dfinity/principal";
+import { sha256 } from "js-sha256";
+import { SubAccount } from "../account_identifier";
+// import { SubAccount } from "../account_identifier";
+import { asciiStringToByteArray } from "./converter.utils";
+
+// 32 bytes
+export const buildSubAccount = (
```

Comment:
This function will never be used anywhere outside of `stakeNeuron`, so probably simpler to keep it as a private method in the governance canister. I'd also rename it to `buildStakeNeuronSubAccount` to make that more clear.